Engine/DataFeeds/Enumerators/Factories/SubscriptionDataReaderSubscriptionEnumeratorFactory.cs

```diff
@@ -139,7 +149,22 @@ public void Dispose()
 
                 _resultHandler.DebugMessage(message);
             }
-           
+
+            // Log our start date adjustments because of map files
+            if (!_startDateLimitedWarnings.IsNullOrEmpty())
+            {
+                var message = $"The starting dates for the following symbols have been adjusted to match their" +
+                    $" map files first date: {string.Join(", ", _numericalPrecisionLimitedWarnings.Values.Take(_numericalPrecisionLimitedWarningsMaxCount))}";
```

Comment:
Copy paste typo `{string.Join(", ", _numericalPrecisionLimitedWarnings.Values.Take(_numericalPrecisionLimitedWarningsMaxCount))}`

Reply:
😮 Thank you!!